tools/gromacs/sim.xml

```diff
@@ -98,16 +98,28 @@
                     pme_order      = 4        ; cubic interpolation
                     fourierspacing  = 0.16    ; grid spacing for FFT
                     ; Temperature coupling is on
+                    #if $sets.mdp.couplinggroups.wholesystem =="true":
+                    tcouple   = V-recale ; modified Berendsen thermostat
+                    tc-grps = System ; coupling the whole system. Can be used in any system, beyond peptides. 
+                    tau_t    = 0.1 ; time constant in ps
```

Comment:
```suggestion
                    tau_t    = 0.1 ; time constant in ps
```
I guess this can go outside the `if` as well

Reply:
I wouldn't, because there is one tau_t value when you're single coupling, and two values when dual coupling (protein non-protein), so having this single value pertains to this "if" block specifically

Reply:
yes, missed that, you are right